lib/container/client.go

```diff
@@ -0,0 +1,59 @@
+package container
+
+import (
+	"fmt"
+	"os/exec"
+
+	credClient "github.com/docker/docker-credential-helpers/client"
+)
+
+var (
+	client   Client
+	binaries = []RuntimeBinary{
+		RuntimeBinaryNerdctl,
+		RuntimeBinaryPodman,
+		RuntimeBinaryDocker,
```

Comment:
we might want to make docker come first, the `hof` action failure, while not because of this, did illuminate that we are using docker in the action, but workers have both docker and podman installed by default. So hof itself is picking up on this extra podman we were not aware of, and thus some things are being done with docker while others with podman. Docker is probably still the easiest to have as the default.